Tests/NIOPosixTests/ChannelTests.swift

```diff
@@ -2770,39 +2770,66 @@ public final class ChannelTests: XCTestCase {
         XCTAssertEqual(1, counter.errorCaughtCalls)
     }
 
-    func testTCP_NODELAYisOnByDefault() throws {
+    func _testTCP_NODELAYis(
+        enabledByDefault: Bool,
```

Comment:
This kind of "narrative style" function name is discouraged in Swift APIs, and in this particular instance I think it's hard to read. I think it'd be better to rename this to `_testTCP_NODELAYDefaultValue(value:`

Reply:
Done.